modules/api/php/views/project_0_0_4_dev.class.inc

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+/**
+ * PHP Version 7
+ *
+ * @category ApiViews
+ * @package  Loris
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\api\Views;
+
+/**
+ * This class formats a Project object into arrays following the API
+ * specifications.
+ *
+ * @category ApiViews
+ * @package  Loris
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class Project_0_0_4_Dev extends Project
+{
+    /**
+     * Produce an array representation of this project.
+     *
+     * @return array
+     */
+    public function toArray(): array
+    {
+        $meta = ['Project' => $this->_project->getName()];
+
+        return [
+            'Meta'        => $meta,
+            'Subprojects' => array_values(
+                \Utility::getSubprojectList(
+                    $this->_project->getId()
+                )
+            ),
```

Comment:
I think this key should be removed in order to decouple this from #7747 and unblock it. The rest of the results in this return seem uncontroversial and are all things that already exist in the API. Since it's still `-dev`, it's easier to add something to it if the other moves forward than remove it.

Reply:
The whole point of this PR is to add that key.

Reply:
In that case I don't see why it's a separate PR. The whole point of the key seems to be to make the other one accessible.

Reply:
Because at first, the subporjects were at the root of the api (like sites and projects and candidates)